### ISSUE TITLE
Observers not detecting random ticks properly on saplings 

### ISSUE THREAD

**Issue by masterRevillo** [PaperSpigot/Paper#7872]
### Expected behavior

Per vanilla behavior, I expect an observer to detect every time a sapling is random-ticked. Regardless of whether or not a tree grows from a sapling, the observer should detect and output a redstone signal

### Observed/Actual behavior

Observers seem to only detect the first time the sapling is random-ticked. It outputs a signal for the first tick, but nothing afterwards. 

### Steps/models to reproduce

- Place a sapling down on dirt. Ensure it has enough light to grow. 
- Obstruct the sapling from growing. 
- Place an observer facing the sapling, and run a redstone line out behind it
- Spam the sapling with bone meal
- Notice that only a single pulse is emitted from the observer, once, and then never again

https://user-images.githubusercontent.com/14932951/172025868-3180d2e2-007a-4c95-845c-9f7279ccff20.mp4

### Plugin and Datapack List

Plugins (2): floodgate, Geyser-Spigot

There are 16 data packs enabled: [vanilla (built-in)], [file/afk display.zip (world)], [file/customizable armor stands v2.4.0.zip (world)], [file/double slab crafting.zip (world)], [file/kill cod v1.0.0.zip (world)], [file/player head drops v1.0.0.zip (world)], [file/spawning spheres v1.0.0.zip (world)], [file/armour statues v2.7.1.zip (world)], [file/coordinates hud v1.1.2.zip (world)], [file/multiplayer sleep v1.0.2.zip (world)], [file/track raw statistics v1.3.0.zip (world)], [file/nether portal coords v1.1.0.zip (world)], [file/larger phantoms v1.2.1.zip (world)], [file/more mob heads v2.9.1.zip (world)], [file/disable-dragon-griefing.zip (world)], [file/bukkit (world)]

### Paper version

This server is running Paper version git-Paper-265 (MC: 1.18.2) (Implementing API version 1.18.2-R0.1-SNAPSHOT) (Git: 993f828)

**Comment by Machine-Maker** [comment]
Duplicate of https://github.com/PaperMC/Paper/issues/6458

**Comment by masterRevillo** [comment]
Should I expect this to be fixed in the latest version of Paper? I downloaded git-Paper-372 and am still experiencing the same issue.

**Comment by Machine-Maker** [comment]
No its not fixed. See the issue I linked. There is an open PR that fixes it however. So it should be fixed at some point.

**Comment by lynxplay** [comment]
No, your issue was merely closed because we are already tracking this issue. This will be fixed once #6464 is merged.

**Comment by masterRevillo** [comment]
My mistake, I thought the PR was already merged
